package.json

```diff
@@ -107,20 +108,20 @@
     "@guardian/ab-core": "^2.0.0",
     "@guardian/ab-react": "^2.0.1",
     "@guardian/consent-management-platform": "^9.0.0",
-    "@guardian/libs": "^3.3.0",
-    "@guardian/source-react-components-development-kitchen": "^0.0.19",
-    "@guardian/src-accordion": "^3.10.1",
-    "@guardian/src-brand": "^3.10.1",
-    "@guardian/src-button": "^3.10.1",
-    "@guardian/src-checkbox": "^3.10.1",
-    "@guardian/src-foundations": "^3.10.1",
-    "@guardian/src-icons": "^3.10.1",
-    "@guardian/src-layout": "3.10.1",
-    "@guardian/src-link": "^3.10.1",
-    "@guardian/src-radio": "^3.10.1",
-    "@guardian/src-text-input": "^3.10.1",
+    "@guardian/libs": "^3.5.0",
+    "@guardian/source-react-components-development-kitchen": "^0.0.20",
+    "@guardian/src-accordion": "^3.10.2",
+    "@guardian/src-brand": "^3.10.2-alpha.1",
+    "@guardian/src-button": "^3.10.2",
+    "@guardian/src-checkbox": "^3.10.2",
+    "@guardian/src-foundations": "^3.10.2-alpha.1",
+    "@guardian/src-icons": "^3.10.2",
+    "@guardian/src-layout": "3.10.2-alpha.1",
```

Comment:
we shouldn't be bringing alpha packages into production, i suggest leaving this at `3.10.1` for now